02-Git/Makefile

```diff
@@ -12,4 +12,5 @@ clean:
 	-rm ./$(BUILD_DIR)/$(TARGET)
 
 run:
-	./$(BUILD_DIR)/$(TARGET)
```

Comment:
Please do not do fixes to not merged patches!
Squash this commit with the first one.